vuu-ui/packages/vuu-layout/src/stack/Stack.tsx

```diff
@@ -146,6 +146,7 @@ export const Stack = forwardRef(function Stack(
           allowDragDrop={TabstripProps.allowDragDrop !== false}
           animateSelectionThumb
           className={cx("vuuTabHeader", tabstripClassName)}
+          aria-label="data tabs"
```

Comment:
I was trying to avoid introducing things like `data-testid` that are only used for testing - I wanted the tests to reflect how the app is used. It made sense for a user to be looking for an element with the "tablist" role, but I needed an intuitive way to distinguish the 'data' tabs from the tabs in the left nav. Do you have any thoughts on a more appropriate name, or a better approach to isolate the element?